docs/editor_manual/new_pages/inserting_videos.md

```diff
@@ -4,11 +4,11 @@
 
 It is possible to embed media into the body text of a web page by clicking the _Embed_ button in rich text toolbar.
 
-![](../../_static/images/screen20_insert_video_form.png)
+!["Insert embed" modal dialog, with a single URL field and an Insert button under](../../_static/images/screen20_insert_video_form.png)
```

Comment:
`docs/_static/images/screen20_insert_video_form.png`

We may need to raise a bug for this missing margin above.

![](https://raw.githubusercontent.com/wagtail/wagtail/77e6505f97a819ee53b919ed3ea14a8622e2c217/docs/_static/images/screen20_insert_video_form.png)

Reply:
@lb- isn't this #9130 / #9159 ?

Reply:
No sorry - this is a different issue, the one you mentioned is where we use the tabs element but do not render any tabs. 

The embed modal must be a different structure (no tabs container at all).

Reply:
Filed in https://github.com/wagtail/wagtail/issues/9182.